src/SupervisionSuccess/components/Chart/Chart.jsx

```diff
@@ -0,0 +1,146 @@
+            acc.min = Math.min(baseline, totalPopulation);
+          }
+          return acc;
+        },
+        {
+          chartData: {
+            labels: [],
+            datasets: [
+              {
+                label: "baseline",
+                data: [],
+                borderColor: "#ee3007",
+                backgroundColor: "#ee3007",
+                fill: false,
+              },
+              {
+                label: "totalPopulation",
+                data: [],
+                borderColor: "#2b649c",
+                backgroundColor: "#2b649c",
+                fill: false,
+              },
+            ],
+          },
+          min: Infinity,
+          max: -Infinity,
+        }
+      ),
+    [data]
+  );
+
+  const chartOptions = useMemo(
+    () => ({
+      legend: {
+        display: false,
+      },
+      scales: {
+        yAxes: [
+          {
+            gridLines: false,
+            ticks: {
+              padding: 20,
+              fontSize: 16,
+              suggestedMax: max + (max - min) * VERTICAL_OFFSET,
+              suggestedMin: min - (max - min) * VERTICAL_OFFSET,
+            },
+          },
+        ],
+        xAxes: [
+          {
+            gridLines: false,
+            ticks: { fontSize: 16, padding: 20 },
+          },
+        ],
+      },
+      tooltips: {
+        intersect: false,
+        mode: "index",
+        backgroundColor: "#091e32",
+        titleFontSize: 14,
+        titleAlign: "center",
+        footerFontStyle: "400",
+        footerAlign: "center",
+        caretSize: 0,
+        xPadding: 20,
+        yPadding: 10,
+        callbacks: {
+          title: ([baseline, totalPopulation]) =>
+            `${totalPopulation.value - baseline.value} people`,
+          label: () => null,
+          footer: ([baseline]) => `${baseline.label} months`,
+        },
+      },
+    }),
+    [min, max]
+  );
+
+  const drawLinePlugin = {
+    afterDraw: function (chart) {
+      if (chart.tooltip._active && chart.tooltip._active.length) {
+        const [firstPoint, secondPoint] = chart.controller.tooltip._active;
+        const topY = Math.max(firstPoint._model.y, secondPoint._model.y);
+        const bottomY = Math.min(firstPoint._model.y, secondPoint._model.y);
+        if (topY === bottomY) return;
+        const x = firstPoint._model.x;
+        const ctx = chart.ctx;
+        ctx.save();
+        ctx.beginPath();
+        ctx.moveTo(x, topY - 5);
+        ctx.lineTo(x, bottomY + 5);
+        ctx.setLineDash([10, 2]);
+        ctx.lineWidth = 1;
+        ctx.strokeStyle = "#07aded";
+        ctx.stroke();
+        ctx.restore();
+      }
+    },
+  };
+
+  return (
+    <div className="chart">
+      <div className="chart_heading">Total people in prison projected in years</div>
+      <div className="chart_chart">
+        <Line
+          data={chartData}
+          options={chartOptions}
+          plugins={[drawLinePlugin]}
+          width={560}
```

Comment:
With the hard-coded dimensions, the chart is not particularly responsive, but then again the entire component is not responsive yet. We can handle that in a future task but I want to call it out so it's top of mind.